### PR TITLE
Upgrade to EasyRSA 3.x

### PR DESCRIPTION
**Pull request by nikaro**
- Automatically grab the latest version of EasyRSA 3.x branch
- Adapt commands for CA and server certificates generation
- Rename vars from key_* to cert_* for those whose were not key related
- Keep default values if vars are empty
- Does not fail on empty MySQL root password, it is bad but it can happen

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -42,16 +41,15 @@ fi
...
 
-read -p "Server ip: " ip_server
+read -p "Server Hostname/IP: " ip_server
 
-read -p "Port [default: 443]: " server_port
+read -p "Port [443]: " server_port
 
-if [[ "$server_port" == "443" || "$server_port" == "" ]]; then
+if [[ -z $server_port ]]; then
   server_port="443"
```
Comment (SansPseudoFix) [inline install.sh]: Oh yep, definitely better than my version :+1:

File: install.sh [hunk 2]
```diff
@@ -88,76 +81,100 @@ fi
...
 # TODO MySQL port & host ?
 
 
 printf "\n################## Certificates informations ##################\n"
-key_size="0"
 
-while [ "$key_size" != "1024" -a "$key_size" != "2048" -a "$key_size" != "4096" ]; do
-  read -p "Key size (1024, 2048 or 4096): " key_size
-done
+read -p "Key size (1024, 2048 or 4096) [2048]: " key_size
```
Comment (Chocobozzz) [inline install.sh]: Why did you remove the loop that checked the validity of the keysize?

**Comment by Chocobozzz** [comment]
Nice, I just have a little question (in code comment). Thanks @nikaro :)

File: install.sh (hunk 2 above)
Comment (nikaro) [inline install.sh]: Because with EasyRSA 3 there is a default value, which is 2048, and the loop doesn't allow to just keep it by pressing Enter.
